packages/quantic/force-app/examples/main/cspTrustedSites/QuanticCoveoCloudPlatform.cspTrustedSite-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CspTrustedSite xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
I added it to the examples package so the Coveo for Salesforce package does not have to be installed.

We can move the CSP to the main package, but it might get annoying. I guess most implementations will need the Coveo for Salesforce package to generate search tokens. So we'd end up having 2 CSPs for Coveo Cloud.

Reply:
I actually don't know what most people will do for the token. I think most current users just used our template for the custom token generator. That's not a lot of users though. In any case, i think you're right that it doesn't need to be done right now.